### DISCUSSION TITLE
configuring-named-options-using-iconfigurenamedoptions-and-configureall/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#106] [Announcements]
# configuring-named-options-using-iconfigurenamedoptions-and-configureall/

In this post I look at how to configure named options that require injected services and how to configure all named options at once using ConfigureAll()

https://andrewlock.net/configuring-named-options-using-iconfigurenamedoptions-and-configureall/

**Comment by zipif** [comment]
hi
can u please show how to create mock of specific named options?
i try Options.Create() ,but there is no parameter for name
thanks

[a comment by an automated account is not shown]
